_posts/en/newsletters/2019-12-04-newsletter.md

```diff
@@ -0,0 +1,155 @@
+  behalf of LN nodes that may be offline, recovering any funds that
+  would otherwise have been lost due to an old channel state being
+  confirmed onchain.  The goal of the proposed specification is to allow
+  all LN implementations to interoperate with any watchtower rather than
+  there being a different watchtower implementation for every LN
+  implementation.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #3259][] add experimental support for payment secrets,
+  which are proposed in [BOLTs #643][] as part of enabling [multipath
+  payments][topic multipath payments].  The payment secret is generated
+  by the receiver and included in their [BOLT11][] invoice.  The spender
+  includes this secret in the part of their payment that's encrypted to
+  the receiver's key.  The receiver only accepts an incoming payment for
+  the invoice if it contains the secret, preventing any other nodes from
+  probing the receiver to see whether it's expecting additional payments
+  to a previously-used payment hash.
+
+- [C-Lightning #3268][] changes the default network from Bitcoin testnet
+  to Bitcoin mainnet.  It also adds a new configuration option,
+  `include`, that includes any configuration directives provided in the
+  indicated file.
+
+{% include linkers/issues.md issues="3259,643,3268" %}
+[news48 coshv]: /en/newsletters/2019/05/29/#proposed-new-opcode-for-transaction-output-commitments
+[news49 stb]: /en/newsletters/2019/06/05/#coshv-proposal-replaced
+[oconnor checksig pos]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-November/017495.html
+[wuille safer sighashes]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2018-November/016508.html
+[irc checksig pos]: http://www.erisian.com.au/taproot-bip-review/log-2019-11-28.html#l-65
+[towns checksig pos]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-November/017497.html
+[zmn composable musig]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-November/017493.html
+[nodelets proposal]: https://lists.linuxfoundation.org/pipermail/lightning-dev/2019-October/002236.html
+[news74 taproot updates]: /en/newsletters/2019/11/27/#schnorr-taproot-updates
+[ctv post]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-November/017494.html
+[bip-ctv]: https://github.com/JeremyRubin/bips/blob/ctv/bip-ctv.mediawiki
+[ctv workshop]: https://forms.gle/pkevHNj2pXH9MGee9
+[oconnor state variable]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-November/017496.html
+[oconnor suggested amendments]: https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-May/016973.html
+[irc ctv]: https://freenode.irclog.whitequark.org/bitcoin-wizards/2019-11-28#25861123;
```

Comment:
The url without the trailing semi-colon appears to display a cleaner log without the background activity pollution.

(FWIWI I found this to be a really interesting discussion; good choice to mention it this week.)

Reply:
> The url without the trailing semi-colon appears to display a cleaner log without the background activity pollution.

Are you sure?  Looks the same to me on Firefox (with and without JS) and Chromium.

Reply:
Chalk it up to brainfart of the day. Among other browser extensions, I have NoScript installed which I use to block all JS by default. I don't visit many sites and so authorise individual scripts on a site-by-site, script-by-script basis. So here I overlooked that both JS sources in the URL were blocked, and the non-semi-colon version was displaying the page from a bit lower down, where Gregory begins writing, and without the cruft at the top. It was time to take a break from the computer :D